### PR TITLE
fix: add support for skipAudits used in isolation

### PR DESCRIPTION
**Pull request by patrickhulce**

### CODE REVIEW

File: lighthouse-core/config/config.js [hunk 1]
```diff
@@ -381,9 +381,13 @@ class Config {
    * @param {!Array<string>=} skipAuditIds
    * @return {!Object<string, {audits: !Array<{id: string}>}>}
    */
-  static filterCategoriesAndAudits(oldCategories, categoryIds = [], auditIds = [],
-      skipAuditIds = []) {
+  static filterCategoriesAndAudits(oldCategories, categoryIds, auditIds, skipAuditIds) {
```
Comment (ebidel) [inline lighthouse-core/config/config.js]: doc string need updating now.

Comment (patrickhulce) [inline lighthouse-core/config/config.js]: why? they're still optional

**Comment by patrickhulce** [comment]
@ebidel I'm merging into the branch that added the test that's currently failing :)

**Comment by paulirish** [comment]
ok let's merge this and in that PR we can decide if we want the functionality. :)
thx

File: lighthouse-core/config/config.js (hunk 1 above)
Comment (ebidel) [inline lighthouse-core/config/config.js]: True. Not sure what the MO is for closure docs. IIRC, if you don't specify a default arg in the arg list, the var needs to be prefixed with `opt_`. That might be outdated though. cc @brendankenny

Comment (brendankenny) [inline lighthouse-core/config/config.js]: we haven't required it and the compiler doesn't really care anymore. It is nice to document in lieu of having the explicit `opt_` in the param name, though

**Pull request merged**